README.md

```diff
@@ -1,3 +1,52 @@
 # State of Our Networks
 
-This repository has organizing materials for Our Networks team.
+This repository has organizing materials for the Our Networks team.
+
+## Infrastructures
```

Comment:
Unsure about this and whether needed as a separate header. Also I would split apart most of these into docs and link from the readme.